runtime.txt

```diff
@@ -0,0 +1 @@
+python-2.7.14
```

Comment:
I wonder why do we need to specify this manually now. When deploying to production, I see this:
```
The latest version of Python 2 is python-2.7.14 (you are using python-2.7.12, which is unsupported).
We recommend upgrading by specifying the latest version (python-2.7.14).
```

Reply:
Unfortunately, I can't add any comments to this file :(
Python3 is the default python for the new apps - that probably doesn't affect existing production but it's breaking deployment of new apps if you don't specify runtime explicitly.
Related docs: https://devcenter.heroku.com/changelog-items/1169